build-logic/src/main/kotlin/org.sonarsource.iac.java-conventions.gradle.kts

```diff
@@ -41,7 +41,7 @@ tasks.jacocoTestReport {
     reports {
         xml.required.set(true)
         csv.required.set(false)
-        html.required.set(false)
+        html.required.set(providers.environmentVariable("CI").map(String::toBoolean).orElse(true))
```

Comment:
Shouldn't it be false when we're in ci?
`html.required.set(!providers.environmentVariable("CI").isPresent)`

Reply:
Yes, good catch, would be a silly mistake :sweat_smile: